developer_manual/app/changelog.rst

```diff
@@ -0,0 +1,39 @@
+=========
+Changelog
+=========
+
+.. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
+
+The following changes went into ownCloud 8:
+
+
+Breaking changes
+================
+
+* Setting session variables inside a controller requires the **@UseSession** annotation, otherwise the session will be closed
+
+Deprecations
+============
+
+.. note:: Deprecations on interfaces also affect the implementing classes!
```

Comment:
I sent an email about some namespace fixes for some classes and constants in nov or dec on the devel list.
That should be added in short words aswell